RELEASENOTES.md

```diff
@@ -1,2 +1,3 @@
 - `generate-script` command in `ado2gh` and `gh gei` will no longer generate an empty script if no migratable repos were found.
 - `--ado-pipeline` arg in the `ado2gh rewire-pipeline` command will now accept a pipeline name without the full pipeline path, so long as there is only one pipeline found that matches that name. If there are multiple pipelines with the same name (in different pipeline folders), you will need to provide the full pipeline path.
+- Fixed a bug where the `migrate-repo` command would return `301` for a recently renamed repo. 
```

Comment:
not sure exactly the wording I would use, but I would be less specific about `301` codes. Just that it used to error, and now it doesn't.

Reply:
addressed in cf9038cb5c8dde33e99996ccc5c494e1406381cf